react/demo/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
-	"extends": "./tsconfig.check.json",
+	"extends": "../tsconfig.json",
+	"include": ["src"],
```

Comment:
Why is this change needed?
By the way, this is not related to the topic of this PR.

Reply:
Indeed, my bad. Will remove this.